### ISSUE TITLE
Proposal: Reviewer of Truth in Marketing

### ISSUE THREAD

**Issue by njordhov** [blockstack/app-mining#155]
**What is the problem you are seeing? Please describe.**
 
*High ranking apps benefit from misleading marketing.*

**How is this problem misaligned with goals of app mining?**

It incentivizes falsehood and hyperbole over creating quality apps that preserve individual’s fundamental digital rights, with developers getting rewarded without having to deliver on promises.

**What is the explicit recommendation you’re looking to propose?**

*Add a reviewer for truth in marketing.*

The reviewer may penalize the score of apps found to practice misleading marketing. The penalty could be a score multiplier from 0 to 1, where 0 is pants-on-fire.

Initially the focus could be the Product Hunt presentation at time of hunt, eventually also the app itself and its landing page as recorded by TryMyUi. The community could be invited to submit problematic marketing claims for evaluation.

**Describe your long term considerations in proposing this change. Please include the ways you can predict this recommendation could go wrong and possible ways mitigate.**

For scale and feasibility, consider only reviewing the top ranking apps for truthful marketing, with a cutoff limiting reviews to apps that would receive a substantial payout.

Benefit: The actual apps developed may live more up to how they are presented in marketing. Consumers would have more reason to trust the Blockstack ecosystem in being truthful and deliver on our promises.

Cost: May lead to reduced attention from media interested in sensationalist reporting and click-bait headlines.

**Additional context**

#98 on anti-patterns that might decrease the ranking if used.

**Comment by cuevasm** [comment]
Hey thanks for the suggestion! I'll get into it: For one, it's against the general spirit of App Mining to punish bad behavior over incentivizing good. More problematic is getting into punishment in situations like this because it's relatively easy for someone to purchase bad votes, mentions, make articles, etc and point them at someone else's app that they want to see punished. 

I also see only reviewing the top apps as very problematic and unfair, this would simply incentive people to try and fly under the radar here. 

It's also one of those things that's completely untenable to enforce or prove when you actually start looking into the data. With Awario we saw that everyone has a very different opinion of what's acceptable or considered 'truth in marketing' and auditing is time consuming and uneven (hence the change).

In short, the biggest problem here is there's no objective or standard way to measure truth or to determine what is misleading or not with any consistency. 

I generally think App Miners should have 'freedom of speech' and reap the benefits or consequences of their marketing choices just like normal businesses do.

**Comment by njordhov** [comment]
Truthful marketing is less of an issue with Product Hunt no longer being a reviewer. The issue should be reopened if/when marketing again becomes a consideration in app mining.

**Comment by njordhov** [comment]
Now that app mining as we know it is terminated, truth in marketing should be included in the CBE ethos. Users should have reason to trust the Blockstack ecosystem in being truthful and deliver on our promises.
